docs/snippets/react/documentscreen-story-msw-graphql-query.js.mdx

```diff
@@ -7,11 +7,10 @@ import { ApolloClient, ApolloProvider, InMemoryCache } from '@apollo/client';
 
 import { graphql } from 'msw';
 
-import DocumentScreen from './DocumentScreen';
+import DocumentScreen from './YourPage';
```

Comment:
Any reason for this update? 😄

Reply:
Indeed it is. It's misleading, for context see [here](https://storybook.js.org/docs/react/workflows/build-pages-with-storybook#args-composition-for-presentational-screens), we're naming it "YourPage.js" and it transforms in the mocking with [MSW section](https://storybook.js.org/docs/react/workflows/build-pages-with-storybook#mocking-rest-requests-with-msw-addon). As I was checking this I spotted it and adjusted the import accordingly for consistency.

Reply:
Awesome, thank you! That makes a lot of sense.